src/main/java/tuteez/logic/commands/AddCommand.java

```diff
@@ -26,25 +26,27 @@ public class AddCommand extends Command {
     public static final String COMMAND_WORD = "add";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a student to the address book. "
-            + "Parameters: "
```

Comment:
Nice work with changing this MESSAGE_USAGE which makes it easier for users to know what are compulsory and optional parameters! Maybe we can find ways to make this more concise to fit within the result display without needing the users to scroll? Let's discuss this again!